sympy/core/numbers.py

```diff
@@ -161,7 +161,7 @@ def igcd(*args):
             a = _gcdcache[(a, b)]
         except KeyError:
             a, b = as_int(a), as_int(b)
```

Comment:
If anything, write `t = as_int(a), as_int(b)` here. But, again, I don't see why this change is necessary. It just seems to create an unnecessary variable. Is creating the tuple slow (but the way you've got it, you still create the tuple twice)?